cmd/kube-apiserver/tools.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	// enforce dependency on our openshift libs
+	_ "github.com/openshift/apiserver-library-go/pkg/admission/imagepolicy"
```

Comment:
@marun do we still need this with the new rebase procedure?

Reply:
Dropped.